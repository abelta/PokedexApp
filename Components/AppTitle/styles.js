import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  appTitle: {
    color: 'white',
    fontSize: 19,
    fontWeight: 'bold',
  },
});

export default styles;
