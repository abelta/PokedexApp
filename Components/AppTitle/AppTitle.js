import React from 'react';
import { Text } from 'react-native';
import styles from './styles';

const AppTitle = () => <Text style={styles.appTitle}>Pok√©dex</Text>;

export default AppTitle;
