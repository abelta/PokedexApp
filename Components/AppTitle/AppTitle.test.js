import React from 'react';
import renderer from 'react-test-renderer';
import AppTitle from '.';

describe('AppTitle', () => {
  it('should render properly', () => {
    const tree = renderer.create(<AppTitle />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
