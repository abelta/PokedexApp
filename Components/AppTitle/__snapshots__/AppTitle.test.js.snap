// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTitle should render properly 1`] = `
<Image
  source={1}
  style={
    Object {
      "height": 30,
      "width": 170,
    }
  }
/>
`;
